azkaban-common/src/test/java/azkaban/jobtype/JobTypeManagerTest.java

```diff
@@ -53,6 +59,20 @@ public JobTypeManagerTest() {
 
   @Before
   public void setUp() throws Exception {
+    // TODO: reallocf Remove injector creation/deletion when ProcessJob is fully guicified
+    final Props props = new Props();
+
+    final Injector injector = Guice.createInjector(new AbstractModule() {
```

Comment:
This was included in order to handle the partial guicing of the ProcessJob class where I leverage the SERVICE_PROVIDER instead of injecting it the right way.

The problem with doing it the right was is then I have to refactor every class that extends the ProcessJob and then refactor every test for the classes that extend the ProcessJob and then suddenly I'm refactoring another 10+ files. I'd rather break this up into separate PR's which is why I'm leveraging this bandaid for now. Do you think it'd be better to implement Guice throughout this dependency chain now?